python/cudf/tests/test_groupby.py

```diff
@@ -462,7 +462,7 @@ def test_advanced_groupby_levels():
 def test_empty_groupby(func):
     pdf = pd.DataFrame({'x': [], 'y': [], 'z': []})
     gdf = cudf.from_pandas(pdf)
-    assert_eq(func(pdf), func(gdf))
+    assert_eq(func(pdf), func(gdf), check_index_type=False)
```

Comment:
Does Pandas have funky behavior here or do we?

Reply:
This is the issue where since the index is empty, the default dtype is float64 so our index is a `Float64Index`.